protocols/imap/src/main/java/org/apache/james/imap/api/message/response/StatusResponse.java

```diff
@@ -152,6 +154,13 @@ public static ResponseCode condStore(IdRange[] failedRanges) {
             return new ResponseCode("MODIFIED", Arrays.asList(new String[] { failed}), 0, false);
         }
         
+        /** RFC4551 <code>Conditional STORE</code> response code */
```

Comment:
Not at all.
Previously, we used IdRange to store message sequence numbers or uids.
It's why you could live with a single condStore method. The new one is just a copy/paste of the previous that takes the new UidRange type. There's no logic change.